orange3/si.yaml

```diff
@@ -13046,89 +13046,89 @@ widgets/visualize/owmosaic.py:
             \    "
-        : null
+        : false
     class `ProbabilitiesDotItem`:
         def `get_tooltip_text`:
-            'Total: {} <br/>Probability: {:.0%}': null
+            'Total: {} <br/>Probability: {:.0%}': 'Skupno: {} <br/>Verjetnost: {:.0%}'
     class `DiscreteMovableDotItem`:
         def `get_tooltip_text`:
-            'Points: {}': null
-            '{}: {:.0%}<br/>': null
+            'Points: {}': 'Točke: {}'
+            '{}: {:.0%}<br/>': false
         def `_get_tooltip_labels_with_percentages`:
-            <: null
-            '&lt;': null
+            <: false
+            '&lt;': false
     class `GraphicsColorAnimator`:
         def `__init__`:
-            brushColor: null
+            brushColor: false
     class `ContinuousItemMixin`:
         def `get_tooltip_text`:
-            'Points: {}': null
-            'Value: {}': null
+            'Points: {}': 'Točke: {}'
+            'Value: {}': 'Vrednost: {}'
     class `ProbabilitiesRulerItem`:
         def `__init__`:
-            Total: null
-            ' ': null
+            Total: "Skupno"
+            ' ': false
     class `OWNomogram`:
-        Nomogram: null
-        ' Nomograms for Visualization of Naive Bayesian': null
-        ' and Logistic Regression Classifiers.': null
-        icons/Nomogram.svg: null
+        Nomogram: true
+        ' Nomograms for Visualization of Naive Bayesian': "Nomogram za prikaz modelov"
+        ' and Logistic Regression Classifiers.': "naivni Bayes ter logistične regresije."
+        icons/Nomogram.svg: false
         class `Inputs`:
-            Classifier: null
-            Data: null
+            Classifier: false
+            Data: false
         class `Outputs`:
-            Features: null
-        scene: null
+            Features: false
+        scene: false
         class `Error`:
-            'Nomogram accepts only Naive Bayes and ': null
-            Logistic Regression classifiers.: null
-        def `__init__`:
-            Target class: null
-            target_class_index: null
-            normalize_probabilities: null
-            Normalize probabilities: null
-            For multiclass data 1 vs. all probabilities do not: null
-            ' sum to 1 and therefore could be normalized.': null
-            scale: null
-            Point scale: null
-            Log odds ratios: null
-            Scale: null
-            Display features: null
-            display_index: null
-            All: null
-            'Best ranked:': null
-            n_attributes: null
-            ' ': null
-            sort_index: null
-            'Rank by:': null
-            cont_feature_dim_index: null
-            'Numeric features: ': null
-            1D projection: null
-            2D curve: null
+            'Nomogram accepts only Naive Bayes and ': "Nomogram sprejema zgolj naivni Bayesov"
+            Logistic Regression classifiers.: "klasifikator ter logistično regresijo."
```

Comment:
ter -> in? Po mojem "ter" uporabiš bolj takrat, ko že imaš "in" in zato potrebuješ veznik, ki veže šibkeje?